src/engine/aws.ts

```diff
@@ -202,7 +202,34 @@ export class Aws extends ApiSector {
       client: this.client,
       schema: {
         path: z.object({
-          name: z.string()
+          name: z.string(),
+          isSTS: z.boolean().optional()
+        }),
+        searchParams: z.object({
+          role_arn: z.string().optional(),
+          role_session_name: z.string().optional(),
+          ttl: z.string().optional(),
+          mfa_code: z.string().optional()
+        }),
+        response: z.any()
+      }
+    });
+  }
+
+  /**
+   * Generate credentials for the path `/aws/sts/:name`
+   *
+   * @link https://developer.hashicorp.com/vault/api-docs/secret/aws#generate-credentials
+   */
+  get stsCredentials() {
+    return generateCommand({
+      method: 'POST',
+      path: '/aws/sts/{{name}}',
+      client: this.client,
+      schema: {
+        path: z.object({
+          name: z.string(),
+          isSTS: z.boolean().optional()
         }),
         searchParams: z.object({
```

Comment:
Since the request method is `POST`, the parameters should be posted in the body of the request.

Reply:
I can review that if not already done!